scripts/annotation.js

```diff
@@ -154,9 +139,45 @@ $(document).ready( ()=> {
                 seadragon.hide();
                 image_div.show();
             }
-            image_div.attr('src', '/images?id=' + image);
+            let imgURL = '/images?id=' + image;
+            $.get(imgURL)
+                .done(() => {
+                    // knowing the image exists
+                    image_div.attr('src', imgURL);
```

Comment:
in javascript, you can set an onerror handler to img's that will run if the src fails
http://stackoverflow.com/questions/92720/jquery-javascript-to-replace-broken-images
Definitely use this instead of fetching the image twice